backend/app/routers/settings.py

```
@@ -50,6 +84,20 @@ def put_indexing_mode(req: IndexingModeRequest) -> IndexingModeResponse:
 
+@router.get("/llm-config", response_model=LlmConfigResponse)
+def get_llm_config() -> LlmConfigResponse:
+    return LlmConfigResponse(**_get_llm_settings())
+
+
+@router.put("/llm-config", response_model=LlmConfigResponse)
+def put_llm_config(req: LlmConfigRequest) -> LlmConfigResponse:
+    try:
+        _set_llm_settings(req.dict())
+        return LlmConfigResponse(**req.dict())
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)